src/bikeRental/index.ts

```diff
@@ -43,13 +43,9 @@ export function createGetBikeRentalStations(argConfig: ArgumentConfig) {
             ids: stationIds,
```

Comment:
Skjer whitelisting automatisk så lenge man har med id-parameteren?

Reply:
Ja, den vil berre hente stations for dei IDane som er definert. Som her: https://api.dev.entur.io/journey-planner/v2/ide/?query=query%20%28%24ids%3A%20%5BString%21%5D%29%20%7B%0A%20%20bikeRentalStations%28ids%3A%20%24ids%29%20%7B%0A%20%20%20%20id%0A%20%20%20%20name%0A%20%20%20%20bikesAvailable%0A%20%20%20%20spacesAvailable%0A%20%20%7D%0A%7D%0A&variables=%7B%0A%20%20%22ids%22%3A%20%5B%0A%20%20%20%20%22YBE%3AVehicleSharingParkingArea%3A368%22%2C%20%20%20%20%20%20%20%20%0A%20%20%20%20%22YBE%3AVehicleSharingParkingArea%3A83%22%0A%20%20%5D%0A%7D